sdk-consumer/index.ts

```diff
@@ -62,6 +76,7 @@ app.post<UploadTrackRequest>(
         res.send(result);
       }
     } catch (e) {
+      console.log(e);
```

Comment:
was this left in intentionally? if so maybe change to console.error

Reply:
Yes, I left this intentionally since the sdk-consumer is a test/dev playground. Will change to console.error tho